storage/postgres/platform.go

```diff
@@ -78,6 +78,29 @@ func (ps *platformStorage) GetAll() ([]types.Platform, error) {
 	return platforms, nil
 }
 
+func (ps *platformStorage) GetByName(names []string) ([]types.Platform, error) {
+	platformDTOs := []Platform{}
+
+	whereClause := ""
+	for index, name := range names {
+		if index != 0 {
+			whereClause += " OR "
+		}
+		whereClause += fmt.Sprintf("name='%s'", name)
```

Comment:
Is this SQL statement safe? The input is from the user so maybe using placeholders is a better idea?